eth/eventhandler/validation.go

```diff
@@ -25,7 +26,7 @@ func (eh *EventHandler) validateOperators(txn basedb.Txn, operators []uint64) er
 		return fmt.Errorf("no operators")
 	}
 
-	if !ssvtypes.ValidCommitteeSize(len(operators)) {
+	if !ssvtypes.ValidCommitteeSize(conversion.LenUint64(operators)) {
```

Comment:
was this a linter error? `ValidCommitteeSize` receives int and `len` returns int, so I don't get why usage of `conversion` is needed here..